Build/Scripts/PackNuget.ps1

```diff
@@ -0,0 +1,74 @@
+Param(
+    [Parameter(Mandatory=$true)]
+    [string] $BuildArtifactsPath,
+    [Parameter(Mandatory=$true)]
+    [string] $OutputPackageFolder,
+    [Parameter(Mandatory=$true)]
+    [string] $RepoName,
+    [Parameter(Mandatory=$true)]
+    [string] $RepoOwner,
+    [Parameter(Mandatory=$true)]
+    [string] $RepoRoot
+)
+
+function GenerateManifest
+(
+    [Parameter(Mandatory=$true)]
+    $PackageId,
+    [Parameter(Mandatory=$true)]
+    $Version,
+    [Parameter(Mandatory=$true)]
+    $Authors,
+    [Parameter(Mandatory=$true)]
+    $Owners
+)
+{
+    [xml] $template = Get-Content "$RepoRoot\Build\ALAppExtensions.template.nuspec"
```

Comment:
Instead of using `$RepoRoot` and hardcoding the path to the template manifest, why don't we simply add a parameter `$ManifestTemplate` or `$ManifestTemplatePath`?